myuw/dao/quicklinks.py

```diff
@@ -20,14 +20,7 @@
 
 def get_quicklink_data(request):
     affiliations = get_all_affiliations(request)
-    data = {'instructor': affiliations['instructor'],
-            'student': affiliations['student'],
-            'sea_student': affiliations['seattle'],
-            'bot_student': affiliations['bothell'],
-            'tac_student': affiliations['tacoma'],
-            'sea_emp': affiliations['official_seattle'],
-            'bot_emp': affiliations['official_bothell'],
-            'tac_emp': affiliations['official_tacoma']}
+    data = {}
```

Comment:
Removed affiliation data from the quicklinks payload as it is available in the store